File: Interviews/GFG/Daily-Problems/2022/October/Day-09/Help-Ishaan.py
import math

class Solution:
    def isPrime(self, N) -> bool:
        if (N <= 1): return False
        for i in range(2, int(math.sqrt(N)) + 1):
            if N % i == 0:
                return False
        
        return True
        
	def NthTerm(self, N):
	    i = 0
	    while i >= 0:
	        if self.isPrime(N + i) or self.isPrime(N - i):
	            return i
	        i += 1
	            
	    return -1

File: Interviews/GFG/Daily-Problems/2022/October/Day-09/Help-Ishaan.cpp
class Solution {
public:
	bool isPrime(int N) {
		if (N <= 1) return false;
		for (int i = 2; i * i <= N; i++) {
			if (N % i == 0) return false;
		}

		return true;
	}
	int NthTerm(int N) {
		for (int i = 0; ; i++) {
			if (isPrime(N + i) or isPrime(N - i)) {
				return i;
			}
		}

		return -1;
	}
};